GroupIS1024/Models/Player.cs

```diff
@@ -1,17 +1,18 @@
 ﻿using System.ComponentModel.DataAnnotations;
 using System.ComponentModel;
+using Newtonsoft.Json;
 
 namespace GroupIS1024.Models
 {
     public class Player
     {
-
+        [JsonProperty("Player Id")]
```

Comment:
See note above.  No need for spaces.  Actually, I don't think you need this annotation at all.  With an invalid JSON field name, it will cause the JSON Parse to fail.